docs/devguide/xtypes.rst

```diff
@@ -568,9 +646,9 @@ Determining Extensibility
 ..
     Sect<16.6.3>
 
-The extensibility annotations determine how a type may be changed and still be compatible.
+The extensibility annotations can explicitly define the :ref:`extensibility <xtypes--extensibility>` of a type.
 If no extensibility annotation is set, the type will default to appendable.
-The default can be changed with the command line option --default-extensibility *type*, where *type* can be final, appendable, or mutable.
+This default can be changed with the :ref:`--default-extensibility <opendds_idl--default-extensibility>` opendds_idl option.
```

Comment:
I think this would be clearer if the middle sentence was something like "If no extensibility annotation is used, the compiler's default will take effect.  Without additional command-line options, the compiler's default is (appendable)."